packages/server/src/service/letter/index.test.ts

```diff
@@ -26,3 +26,64 @@ test('Letter for all states render correctly', async () => {
     expect(letter?.md('html')).not.toContain('****')
   })
 })
+
+test('trimThreeOrMoreLines works as intended', () => {
+  // Remember that some empty lines have spaces over them
+  const input =
+`zero empty lines
+one empty line
+
+two empty lines
+
+
+four empty lines
+
+\t\t
+
+\t\t
+ - First item
+
+
+ - Second item
+
+\t
+              \t
+ - Third item
+
+
+ - Fourth item
+`
+
+  const output =
+`zero empty lines
+one empty line
+
+two empty lines
```

Comment:
I don't understand how this is still here.  It's 3 newlines, but we want 2 newlines only.

Reply:
Small confusion from my part, I thought we wanted 2 empty lines (excluding newlines at the end of sentences), not two consecutive newlines; should be fixed now, thanks!

Reply:
Thanks.  This is why it's important to write out the test case explicitly.  It was hard to understand the expected behavior unless it's explicit.